plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -528,6 +556,7 @@ private String createContainer(String workspaceId,
               .withEntrypoint(toArrayIfNotNull(service.getEntrypoint()))
               .withLabels(service.getLabels())
               .withNetworkingConfig(networkingConfig)
+              .withUser(getUser())
```

Comment:
@garagatyi user will be the same that is running in che-server container. The same env variable CHE_USER_ID is used by both workspace and che-server.